tests/e2e/testfiles/storage_csm_powerscale_alt_vals_1.yaml

```diff
@@ -245,10 +245,10 @@ spec:
     - name: authorization
       # enable: Enable/Disable csm-authorization
       enabled: false
-      configVersion: v1.12.0
+      configVersion: v1.13.0
       components:
         - name: karavi-authorization-proxy
-          image: quay.io/dell/container-storage-modules/csm-authorization-sidecar:nightly
+          image: quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.13.0
```

Comment:
Not going to comment the same thing on every one that follows this, but we should have an answer on which image we want to pull and use for e2e testing before we merge this.